templates/emails/failed-refund-admin.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Output the email header.
+ */
+do_action( 'woocommerce_email_header', $email_heading, $email ); ?>
+
+<p>
+	<?php
+
+	echo esc_html(
+		sprintf(
+			// translators: 1) an order number, 2) the customer's full name, 3) the reason for the refund request failure.
+			_x(
+				'The refund request for order %1$s from %2$s has failed. Reason:  %3$s. The customer was also notified.',
```

Comment:
As elsewhere, I think we can simplify "refund request" to just "refund".
```suggestion
				'The refund for order %1$s from %2$s has failed. Reason:  %3$s. The customer was also notified.',
```